connect/runtime/src/main/java/org/apache/kafka/connect/runtime/errors/LogReporter.java

```diff
@@ -66,12 +66,12 @@ public void report(ProcessingContext context) {
             return;
         }
 
-        if (context.result().success()) {
+        if (!context.failed()) {
```

Comment:
Extra `!` here?

Reply:
actually, that looks correct -- we don't want to log when there is no failure.